index.d.ts

```diff
@@ -1,3 +1,19 @@
+declare namespace decamelize {
+	interface Options {
+		/**
+		Preserve blocks of uppercase characters: `fooBAR` → `foo_BAR`.
+		@default false
+		*/
+		readonly preserveConsecutiveUppercase?: boolean;
+
+		/**
+		Last character of uppercase block starts new section: `fooBARest` → `foo_BA_rest`.
+		@default false
+		*/
+		readonly consecutiveUppercaseNotGreedy?: boolean;
```

Comment:
This was not mentioned in the issue. Why is it even an option? I think this should just be `true` by default.

Reply:
This came to my mind during implementation: How to deal with the last character of an uppercase block? As you seem not to be happy with it: For my usecase there's no need for this option. We can drop it and set the behavior to `true`.

Please tell me how you want to deal with this option.

Reply:
Yes, drop it and make it `true` by default.